vbuild/cli/gen.py

```
@@ -25,16 +25,16 @@ def command(args: Namespace) -> int:
         return 1
 
     package = parse(filepath)
-    if package.pkgname is None:  # pyright: ignore[reportAny]
+    if package.pkgname is None:  # pyright: ignore[reportUnnecessaryComparison]
         raise Exception("pkgname is missing")
 
-    print(f">>> {package.pkgname}: Generating APKBUILD")  # pyright: ignore[reportAny]
+    print(f">>> {package.pkgname}: Generating APKBUILD")
     fail = False
     for type, msg in package.validate():
```

Ruff (0.15.21)

[error] Variable `type` is shadowing a Python builtin

(A001)